airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -83,26 +85,33 @@
   private final JobPersistence jobPersistence;
   private final ConfigRepository configRepository;
   private final JobCleaner jobCleaner;
+  private final TemporalClient temporalClient;
+  private final WorkflowServiceStubs temporalService;
 
   public SchedulerApp(Path workspaceRoot,
                       ProcessBuilderFactory pbf,
                       JobPersistence jobPersistence,
                       ConfigRepository configRepository,
-                      JobCleaner jobCleaner) {
+                      JobCleaner jobCleaner,
+                      TemporalClient temporalClient,
+                      WorkflowServiceStubs temporalService) {
```

Comment:
@jrhizor I'm realizing it's kinda awkward to have both a temporalClient and a temporalService. My inclination is to figure out how to consolidate them down. Probably means renaming the existing TemporalClient => TemporalWorkflowClient and then wrapping WorkflowServiceStubs in TemporalClient. Does that approach seem reasonable to you or would you think about it differently?

Reply:
That's a good idea!

Reply:
Sounds good